utils/chunked_vector.hh

```diff
@@ -325,8 +327,25 @@ size_t chunked_vector<T, max_contiguous_allocation>::external_memory_usage() con
 template <typename T, size_t max_contiguous_allocation>
 chunked_vector<T, max_contiguous_allocation>::chunked_vector(const chunked_vector& x)
         : chunked_vector() {
-    reserve(x.size());
-    std::copy(x.begin(), x.end(), std::back_inserter(*this));
+    auto size = x.size();
+    if (!size) {
+        return;
+    }
```

Comment:
Probably better to avoid. Most vectors will have content, and the rest of the code will work with zero size.

Reply:
ok

Reply:
dropped in v3